docs/milestones.rst

```diff
@@ -3,28 +3,44 @@ Milestones and roadmaps
 =======================
 
 A subset of JupyterHub projects organizes its releases and the
-items associated with the release using GitHub Issues and Milestones.
+items associated with the release using GitHub Issues, Milestones, and in some cases, roadmaps.
+These span from specific, actionable items, to more high-level goals of the project that
+still require discussion.
+
+In general, Milestones contain items that could use immediate help in implementation and design, and
+we'd love community members to take a shot at addressing them. Roadmaps tend to signal more
+abstract ideas, and often require some discussion before distilling them down into specific issues.
+We also encourage comments, questions, ideas in this process! 
```

Comment:
What is a good (one sentence) way of saying "just because it isn't on the roadmap doesn't mean you can't work on it, the roadmap and project are driven by people speaking up about features they would like to see and then contributing to the discussion and implementation of them. There are two caveats to this:
* items that have been marked as "postponed" in the roadmap, please make an effort to find out why it was postponed and then reconsider if you want to restart the discussion about making it happen earlier.
* you want to implement something which is 180degree opposite of what is currently on the roadmap. It is possible but first try and catch up with discussions that lead to the current direction and consider if you want to re-open discussions about this"

One way would be to skip all the caveat stuff and just say "driven by people who want to see stuff implemented, get involved and help shape the direction of the project. The roadmap tries to set a general direction but does not aim to exclude other ideas."

Reply:
what about something like "The JupyterHub roadmaps are run in a 'yes, and' fashion, meaning that they are ideas the community is excited about and would like to pursue further. They are not exclusionary. They are also dynamic - if you have an idea you'd like to propose, please open an issue or a pull request to begin discussion"

re: 180 degree opposite etc, my intuition is that we don't need to explicitly mention this kinda thing right now...I don't want to over-engineer the roadmap, it is intentionally vague after all :-)

Reply:
ok, latest push tries to make this a bit clearer. LMKWYT